target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -160,7 +160,9 @@ ex2700)
 	;;
 f5d8235-v1|\
 f5d8235-v2)
+	ucidef_set_led_default "power" "POWER" "$board:blue:router" "1"
```

Comment:
Mhh, it still don't explain why this LED need to be enabled after boot. To only reason I meant to understood so far, is because it looks nice. Just drop it!

I guess it would be a good idea to ucidef_set_led_default the blue:wired to 0. It should be only used temporary as a workaround for a missing power led. The same is done on other targets/boards in case there is no power led or the power led is not controllable via gpio.